go/engine/track_token.go

```diff
@@ -226,39 +226,25 @@ func (e *TrackToken) storeRemoteTrack(ctx *Context, pubKID keybase1.KID) (err er
 		return errors.New("unexpeceted KID mismatch between locked and unlocked signing key")
 	}
 
-	var linkID libkb.LinkID
-	var sig string
-	var sigid keybase1.SigID
 	sigVersion := libkb.SigVersion(e.arg.Options.SigVersion)
-	switch sigVersion {
-	case libkb.KeybaseSignatureV1:
-		sig, sigid, err = signingKey.SignToString(e.trackStatementBytes)
-		linkID = libkb.ComputeLinkID(e.trackStatementBytes)
-	case libkb.KeybaseSignatureV2:
-		prevSeqno := me.GetSigChainLastKnownSeqno()
-		prevLinkID := me.GetSigChainLastKnownID()
-
-		sig, sigid, linkID, err = libkb.MakeSigchainV2OuterSig(
-			signingKey,
-			libkb.LinkTypeTrack,
-			prevSeqno+1,
-			e.trackStatementBytes,
-			prevLinkID,
-			false, /* hasRevokes */
-			keybase1.SeqType_PUBLIC,
-			false, /* ignoreIfUnsupported */
-		)
-	default:
-		err = errors.New("Invalid Signature Version")
-	}
+	sig, sigID, linkID, err := libkb.MakeSig(
+		signingKey,
+		libkb.LinkTypeTrack,
+		e.trackStatementBytes,
+		false, /* hasRevokes */
+		keybase1.SeqType_PUBLIC,
+		false, /* ignoreIfUnsupported */
```

Comment:
I wonder if we should give ourselves a couple of typedefs for these boolean arguments. That way we don't have to worry about someone adding more callsites in the future but forgetting these helpful boolean param comments :)

Reply:
done